src/app.ts

```diff
@@ -1,11 +1,34 @@
 import express from "express"
 
+import morgan from "morgan"
+import helmet from "helmet"
+import cors from "cors"
+import router from "./routes/signup"
+
 const app = express()
 
+//I use dev because this microservice won't be run in production. We aren'interested
+//in deploying it to the cloud, just in showing it running locally in class.
+//If this were a constantly running production app we would use "combined"
+app.use(morgan("dev"))
```

Comment:
What do you think of receiving a value through a environment variable and if it doesn't exist we just set it to `dev`?